util/codec/bytes.go

```diff
@@ -202,3 +203,34 @@ func reallocBytes(b []byte, n int) []byte {
 	// slice b has capability to store n bytes
 	return b
 }
+
+// ConvertInt64ToBytes converts []int64 to []byte
+func ConvertInt64ToBytes(ints []int64, bitOrder binary.ByteOrder, offset int) ([]byte, error) {
+	result := make([]byte, 0, len(ints)*(8-offset))
+	var resultLen int
+
+	for _, i := range ints {
+		buf := new(bytes.Buffer)
```

Comment:
Sorry for my vague comment. I mean extract to a local function, not here. It's not  a general function. 
The `codec` is a package for encoding SQL key value to KV data.

Extract the function is not for reuse, but for readability. It makes me comfortable if a function can be viewed in just one page.

Please move this function near `builtinChar`, and use lowercase first letter.

Bye the way, this code seems complex. if you know word reverse, here is a simpler example (seems our team dislike "clever optimize", you can just ignore this idea):

```
func convertInt64ToBytes(ints []int64) []byte {
	var buf bytes.Buffer
	for i := len(ints) - 1; i >= 0; i-- {
		v := ints[i]
		for {
			buf.WriteByte(byte(v % 256))
			v = v / 256
			if v == 0 {
				break
			}
		}
	}
	return reverseSlice(buf.Bytes())
}
```

Rest looks good to me.